### PR TITLE
A fresh take on the intro document

### PR DESCRIPTION
**Pull request by decentralion**
I think the intro/landing page "What is SourceCred" doc is the single most important doc; we should think about what we want to say here and invest in having a clear message.

Here's one take, based on chats with @wchargin, @hammadj, and @Bex2594.

Paired with @Bex2594

### CODE REVIEW

File: docs/intro/what.md [hunk 1]
```diff
@@ -3,12 +3,28 @@ title: 🔮 What is SourceCred?
...
+## An Economic System
 
-The basic idea is: a contribution earns cred if it is connected to other contributions that earn lots of cred. Contributors are connected to the contributions they've helped with, which means that they also earn cred.
+Once a community is up-and-running with cred, they can begin to issue **grain**. Grain is a community-controlled
+digital currency that is distributed based on cred scores. Thus, a contributor earns grain, and a sponsor can
+support that contributor by buying their grain.
 
-The project's community and maintainers have a lot of influence over this process. They can reconfigure SourceCred weights and parameters. For example, they could assign extra weight to some very important contributions, or decrease the weights of contributions that seem spammy. They can also add heuristics to do this automatically.
+Grain represents your skin in the game within a community. You can use your grain to vote on values, and to set priorities.
```
Comment (s-ben) [inline docs/intro/what.md:20]: >You can use your grain to vote on values, and to set priorities.

This comma is making me twitch...Suggest rewording for flow, unless it breaks the meaning.

>You can use your grain to vote on values and set priorities.

**Comment by decentralion** [comment]
Closing as stale.
